tests/stylus/StylintBearTest.py

```diff
@@ -0,0 +1,159 @@
+import os
+from queue import Queue
+
+from bears.stylus.StylintBear import StylintBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'stylint_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as f:
+        output = f.readlines()
+    return output
```

Comment:
if you want you can immediately return:
```
with open(...) as f:
    return f.readlines()
```
the context manager will exit properly ;)